patterns/dxp/aspect_oriented_programming.md

```diff
@@ -0,0 +1,53 @@
+# Aspect Oriented Programming guide of use
```

Comment:
Suggest:

```
# Aspect Oriented Programming (AOP) guidelines
```

ie. 
- introduce "AOP" acronym before first usage below
- "guidelines" is *more betterer* English than "guide of use"